pkg/stanza/fileconsumer/attrs/attrs.go

```diff
@@ -32,6 +39,28 @@ func (r *Resolver) Resolve(path string) (attributes map[string]any, err error) {
 	if r.IncludeFilePath {
 		attributes[LogFilePath] = path
 	}
+	if r.IncludeFileOwner || r.IncludeFileGroup {
+		var file, fileErr = os.OpenFile(fmt.Sprint(path), os.O_RDONLY, 0000)
```

Comment:
I don't think we need to open a new file. Let's just pass the file into `Resolve` instead of the file name.

Reply:
ok thanks for the review. I will do the change. I did not check that the function calling Resolve already add the file.